requirements/strategies.txt

```diff
@@ -1,3 +1,4 @@
 fairscale>=0.4.5
 deepspeed<0.6.0
 horovod>=0.21.2,!=0.24.0  # no need to install with [pytorch] as pytorch is already installed
+hivemind>=1.0.1; sys_platform == 'linux'
```

Comment:
it works only on linux? if yes, should we add some checks and throw exceptions otherwise?

Reply:
Has been done as suggested by @kaushikb11!